nanoFramework.Graphics.Ili9341.nuspec

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+  <metadata>
+    <id>nanoFramework.Graphics.Ili9341</id>
+    <version>$version$</version>
+    <title>nanoFramework.Graphics.Ili9341</title>
+    <authors>nanoframework</authors>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <license type="file">LICENSE.md</license>
+    <releaseNotes>
+    </releaseNotes>
+    <developmentDependency>false</developmentDependency>
+    <readme>docs\README.md</readme>
+    <projectUrl>https://github.com/nanoframework/nanoFramework.Graphics</projectUrl>
+    <icon>images\nf-logo.png</icon>
+    <repository type="git" url="https://github.com/nanoframework/nanoFramework.Graphics" commit="$commit$" />
+    <copyright>Copyright (c) .NET Foundation and Contributors</copyright>
+        <description>This package includes the nanoFramework.Graphics assembly for .NET nanoFramework C# projects.
+It does embed a managed driver Ili9341. It is aimed to be used for images built with Generic Graphics driver.
+This package requires a target with nanoFramework.Graphics v$nativeVersion$ (checksum $checksum$).</description>
+    <tags>nanoFramework C# csharp netmf netnf nanoFramework.Graphics Ili9341</tags>
+    <dependencies>
+      <dependency id="nanoFramework.CoreLibrary" version="1.14.2" />
+      <dependency id="nanoFramework.ResourceManager" version="1.2.13" />
+      <dependency id="nanoFramework.Runtime.Events" version="1.11.6" />
+      <dependency id="nanoFramework.Runtime.Native" version="1.6.6" />
+      <dependency id="nanoFramework.System.Collections" version="1.5.18" />
+    </dependencies>
+  </metadata>
+  <files>
+    <file src="nanoFramework.Graphics\bin\Release\nanoFramework.Graphics.dll" target="lib\nanoFramework.Graphics.dll" />
```

Comment:
The `\nanoFramework.Graphics.*` files can't be shipped with the drivers, because they'll clash with the ones distributed with the nanoFramework.Graphics NuGet.

Instead, if the drivers require it, each nuget should have declared a dependency for it.
To overcome the chicken and egg problem (because they are all released in the same build pipeline) you have grab the version and use it on the declaration.

See on the logging library how this is handled.
[version](https://github.com/nanoframework/nanoFramework.Logging/blob/b1393dd30076a6e62d6fd4a8852a4a140cfe9692/nanoFramework.Logging.Stream.nuspec#L25)

Reply:
There is no file distributed with the Graphic nuget, this replaces the nuget graphic.

Reply:
@josesimoes idea is to replace the nanoFramework.Graphics nuget so you just have to use the Graphics.YourDriver one and you'll get all in there. Because of the update process we have, those will be updated at the same time as the main one.

Reply:
Got you. My bad. I was misreading this. Was under the impression that the "driver" nuget would be ON TOP of a graphics NuGet (which I'm guessing will pop-up soon from the conversations we've been having towards splitting it).

Reply:
No worry! At least like this everything is more clear!
I made the rest of the adjustments, so if nothing else, I let you merge both the native and this managed PR!